server/tests/auth/test_command_policy.py

```
@@ -0,0 +1,753 @@
+        compiled=re.compile(pattern),
+    )
+
+
+@pytest.fixture(autouse=True)
+def reset_cache():
+    command_policy._cache.clear()
+    yield
+    command_policy._cache.clear()
+
+
+@pytest.fixture()
```

Ruff (0.15.12)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)